Fix formatting, update to latest runtime, Quetoo and modules

AsciiWolf opened the pull request.

AsciiWolf commented on org.quetoo.Quetoo.appdata.xml:
```diff
@@ -43,6 +43,6 @@
   </content_rating>
   <releases>
-    <release version="0.1.0" date="2021-04-08" type="development"/>
+    <release version="0.1.0" date="2022-04-08" type="development"/>
```
The release date is based on the date of the last commit. Not sure about the version number though.

bbhtt commented:
you need the appdata patch from the other PR.

also can you keep the whitespace changes in a seperate commit? Helps when reviewing by commit.

bbhtt commented:
is there any reason to update the app and a dependency? I'd like to avoid them if possible since this is already unmaintained.

AsciiWolf commented:
> you need the appdata patch from the other PR.
> 
> also can you keep the whitespace changes in a seperate commit? Helps when reviewing by commit.

I have updated the AppStream metadata (and fixed the second screenshot that was duplicated in your PR).

Sorry, but I won't keep the whitespace changes in separate commit since it would require me to redo all the changes I already did and also it would bring unnecessary complexity. If you have more time, feel free to do it in your PR.

AsciiWolf commented:
> is there any reason to update the app and a dependency? I'd like to avoid them if possible since this is already unmaintained.

There are actually two reasons. The first one is the "new version dialog" that appears every time the game is launched. The second one are graphical glitches that I have in-game with the old version and I want to test whether there are still the same glitches with more recent (latest "master" - so basically a stable beta since the actual development happens in "develop") Quetoo (and dependencies) version.

AsciiWolf commented:
So, the graphical glitches are sadly still there.

![screenshot](https://github.com/flathub/org.quetoo.Quetoo/assets/2339930/b591ad95-998b-4019-b27a-d97df9bed908)

Just like with your PR.

The "new version available" dialog is also still there. And there are no online dedicated servers displayed, although I am not sure whether Quetoo supports them.

bbhtt commented:
Ok cool, I'll apply your other screenshot fix in my PR and update it.

I'd like the keep the diff minimum (it also allows blame to work later)
